app-frontend/src/app/components/filterPane/filterPane.controller.js

```diff
@@ -348,6 +423,87 @@ export default class FilterPaneController {
         }
     }
 
+    setYearFilterMode(mode) {
+        if (mode !== this.yearFilterMode) {
```

Comment:
I am now getting a linting error in this file `454:21  error  Blocks are nested too deeply (4)  max-depth`. Maybe switch this to `if (mode !== this.yearFilterModer) return;`?